hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerOperationClient.java

```diff
@@ -221,6 +221,33 @@ public ContainerWithPipeline createContainer(HddsProtos.ReplicationType type,
     }
   }
 
+  @Override
+  public ContainerWithPipeline createContainer(ReplicationConfig replicationConfig,
+                                                String owner) throws IOException {
```

Comment:
nit: Please do not format method signature like this. Whenever visibility / return type / method name / other modifiers are changed, we would have to reindent all parameters.